pontoon/sync/tasks.py

```diff
@@ -97,62 +92,61 @@ def sync_project(
 
     log.info("Syncing project {0}.".format(db_project.slug))
 
-    # Sync source strings. We cannot sync sources if locale is specified,
-    # because that would apply any source string changes to the specified locale only.
-    if locale:
-        source_changes = {}
-    else:
-        source_changes = sync_sources(db_project, now, force, no_pull)
-        # Skip syncing translations if we already know there's nothing to sync
-        if not source_changes:
-            project_sync_log.skip()
-            return
+    source_changes = sync_sources(db_project, now, force, no_pull)
+    # Skip syncing translations if no source directory found
+    if not source_changes:
+        project_sync_log.skip()
+        return
 
     # Sync translations
     sync_translations(
-        project_pk,
-        project_sync_log.pk,
+        db_project,
+        project_sync_log,
         now,
+        source_changes.get("source_changed"),
         source_changes.get("added_paths"),
         source_changes.get("removed_paths"),
         source_changes.get("changed_paths"),
         source_changes.get("new_entities"),
-        locale=locale,
         no_pull=no_pull,
         no_commit=no_commit,
-        full_scan=force,
+        force=force,
     )
 
 
 def sync_sources(db_project, now, force, no_pull):
-    # Pull source repository
+    # Pull from source repository
     if no_pull:
-        source_repo_changed = True  # Assume changed
+        source_changed = True
```

Comment:
What's the purpose of this change? The previous name seems more verbose and less likely to be confused with `source_changes`.

If you still want to rename the variable, I'd suggest prepending booleans with is/are/has/have like this:
- `source_repo_changed` -> `has_source_repo_changed`

Reply:
Since the code was built upon the previous changes, this stayed. I will follow  "has_changed" convention that is used elsewhere